recipes/graphviz/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set version = "2.38.0" %}
+
+package:
+  name: graphviz
+  version: {{ version }}
+
+source:
+  fn:  graphviz-{{ version }}.tar.gz
+  url: http://www.graphviz.org/pub/graphviz/stable/SOURCES/graphviz-{{ version }}.tar.gz
+  md5: 5b6a829b2ac94efcd5fa3c223ed6d3ae
+  patches:
+    - gvconfig_libdir.patch
+
+build:
+  number: 0
+  skip: true  # [win]
+  detect_binary_files_with_prefix: True
+
+requirements:
+  build:
+    - cairo  # [linux]
+    - expat  # [linux]
+    - libpng  1.6*  # [unix]
+    - libtiff  # [unix]
+    - libtool  # [linux]
+    - freetype  2.5.*  # [unix]
+    - jpeg  # [unix]
+    - pango  >=1.39.0  # [linux]
+    - pkg-config  # [osx]
+    - zlib  1.2.*  # [unix]
+  run:
+    - cairo  # [linux]
+    - expat  # [linux]
+    - libpng  1.6*  # [unix]
+    - libtiff  # [unix]
+    - libtool  # [linux]
+    - freetype  2.5.*  # [unix]
+    - jpeg  # [unix]
+    - pango  >=1.39.0  # [linux]
+    - zlib  1.2.*  # [unix]
```

Comment:
I should add that there are a few things here that still need to be pinned (e.g. `libtiff`, `jpeg`). Please take a look at the pinning [guide]( https://github.com/conda-forge/staged-recipes/wiki/Pinned-dependencies ).

We might want to decide on a pinning for `cario`. We haven't pinned `expat` before, but it might be time to discuss doing that, as well.